### ISSUE TITLE
Failed to `dotnet new classlib` command in tutorial.

### ISSUE THREAD

**Issue by CeciAc** [dotnet/docs#16323]
@hskwakr commented on [Tue Dec 17 2019](https://github.com/dotnet/docs.ja-jp/issues/1066)

[フィードバックをこちらに入力してください]
## Introduction  

Please let me know if this contents is not an appropriate as issue.  

Please let me know if there is any missing information in this message.  
Also, please let me know if you have any helpful information on this issue.  

## Issue  

I tried to this tutorial:  
https://docs.microsoft.com/ja-jp/dotnet/core/tutorials/creating-app-with-plugin-support

And I failed the `dotnet new classlib -o PluginBase` command:  

```powershell  
PS C:\...\AssemblyLoadContext> dotnet new classlib -o PluginBase  
The template "Class library" was created successfully.

Processing post-creation actions...
Running 'dotnet restore' on PluginBase\PluginBase.csproj...
C:\...\AssemblyLoadContext\PluginBase\PluginBase.csproj : error NU1100: '.NETStandard,Version=v2.0' に対する 'NETStandard.Library (>= 2.0.3)' を解決できま
せん。
  C:\...\AssemblyLoadContext\PluginBase\PluginBase.csproj の復元が
 179.49 ms で失敗しました。

Restore failed.
Post action failed.
Description: Restore NuGet packages required by this project.
Manual instructions: Run 'dotnet restore'
```

## MEMO  

I pushed my code to GitHub:  
https://github.com/hskwakr/AssemblyLoadContext/tree/step1.1  

My sell  
```powershell
PS C:\...\AssemblyLoadContext> $PSVersionTable

Name                           Value
----                           -----
PSVersion                      7.0.0-preview.6
PSEdition                      Core
GitCommitId                    7.0.0-preview.6
OS                             Microsoft Windows 10.0.18362
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

My .NET Core
```powershell
PS C:\...\AssemblyLoadContext> dotnet --info
.NET Core SDK (global.json を反映):
 Version:   3.1.100
 Commit:    cd82f021f4

ランタイム環境:
 OS Name:     Windows
 OS Version:  10.0.18362
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\3.1.100\

Host (useful for support):
  Version: 3.1.0
  Commit:  65f04fb6db

.NET Core SDKs installed:
  3.1.100 [C:\Program Files\dotnet\sdk]

.NET Core runtimes installed:
  Microsoft.AspNetCore.App 3.1.0 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 3.1.0 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]    
  Microsoft.WindowsDesktop.App 3.1.0 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]

To install additional .NET Core runtimes or SDKs:
  https://aka.ms/dotnet-download
```

## Thank you for reading.  

---
#### ドキュメントの詳細

⚠ *このセクションを編集しないでください。 docs.microsoft.com で必須です ➟ GitHub の問題のリンク。*

* ID: 06023407-704f-0ace-e89a-eedd9f95a007
* Version Independent ID: f25a6e1a-cb10-867b-0522-a377bf9d016b
* Content: [プラグインがある .NET Core アプリケーションを作成する - .NET Core](https://docs.microsoft.com/ja-jp/dotnet/core/tutorials/creating-app-with-plugin-support)
* Content Source: [docs/core/tutorials/creating-app-with-plugin-support.md](https://github.com/dotnet/docs.ja-jp/blob/live/docs/core/tutorials/creating-app-with-plugin-support.md)
* Product: **dotnet-core**
* GitHub Login: @jkoritzinsky
* Microsoft Alias: **jekoritz**

---

@srvbpigh commented on [Wed Dec 18 2019](https://github.com/dotnet/docs.ja-jp/issues/1066#issuecomment-566847817)

Hello, @hskwakr
 
Thank you for your feedback.
 
We are actively reviewing your comments and will get back to you soon.

Kind regards,
Microsoft DOCS International Team

**Comment by mairaw** [comment]
Thanks for your feedback @hskwakr.  This issue doesn't seem related to the content in this article.

Have you tried running dotnet restore as the manual instructions stated?

**Comment by hskwakr** [comment]
Thank you for your reply.  
I'm sorry. I didn't explain it enough.
**And I seem the issue has already been solved.**

I had changed the value `TargetFramework` from `netstandard2.0` to `netcoreapp3.1` in the file `PluginBase/PluginBase.csproj`.  
https://github.com/hskwakr/AssemblyLoadContext/commit/5eff8a47fc9ab436d324e31e59043ba2715c2bc2#diff-92b53be55bc10347ef6c4f3f90a23928

And I had tried running `dotnet restore PluginBase`.  
And when I had tried `dotnet restore PluginBase`, I had received below message.

```powershell
AssemblyLoadContext> dotnet restore .\PluginBase\
  C:\msys64\home\akikan\prg\sandbox\CSharp\AssemblyLoadContext\PluginBase\PluginBase.csproj の復元が 32.76 ms で完了しました。
```

I had not believed that it is fixed the issue when I read the message.  
Because I don't know a way to check that those plugins is running correct.

Now, I have different thinking.  
I think If I had no choice but to read the message, then I should believe it.

Thank you for helping me.
